src/DurableSDK/Commands/Internals/SetFunctionInvocationContextCommand.cs

```diff
@@ -52,7 +52,7 @@ protected override void EndProcessing()
                     // De-serialize the orchestration context
                     JsonSerializerSettings serializerSettings = new JsonSerializerSettings
                     {
-                        TypeNameHandling = TypeNameHandling.All
+                        TypeNameHandling = TypeNameHandling.Auto
```

Comment:
I know this is a draft, but I recommend being very careful with this change. I remember spending a fair bit of time trying to figure out which enum value to provide here, and finding that `.All` was safest. But the details escape me so take it with a grain of salt.